curvefs/src/client/fuse_client.cpp

```diff
@@ -751,10 +758,10 @@ CURVEFS_ERROR FuseClient::FuseOpRename(fuse_req_t req, fuse_ino_t parent,
     RETURN_IF_UNSUCCESS(GetTxId);
     RETURN_IF_UNSUCCESS(Precheck);
     RETURN_IF_UNSUCCESS(RecordOldInodeInfo);
-    RETURN_IF_UNSUCCESS(LinkDestParentInode);
     RETURN_IF_UNSUCCESS(PrepareTx);
     RETURN_IF_UNSUCCESS(CommitTx);
-    renameOp.UnlinkSrcParentInode();
+    RETURN_IF_UNSUCCESS(LinkDestParentInode);
+    RETURN_IF_UNSUCCESS(UnlinkSrcParentInode);
```

Comment:
maybe we shouldn't check the result of `UnlinkSrcParentInode`.

Reply:
done